pyvista/plotting/charts.py

```diff
@@ -1025,7 +1032,9 @@ class _Chart(DocSubs):
     def __init__(self, size=(1, 1), loc=(0, 0)):
         super().__init__()
         self._background = _ChartBackground(self)
-        self._x_axis = Axis()  # Not actually used for now (see note in Chart2D), but still present for the
+        self._x_axis = (
+            Axis()
+        )  # Not actually used for now (see note in Chart2D), but still present for the
         self._y_axis = Axis()  # Charts.toggle_interaction code
```

Comment:
This 2-line comment ends up being broken along with the corresponding Python code.